datasets/us/ky/med_exclusions/us_ky_med_exclusions.yaml

```diff
@@ -0,0 +1,69 @@
+title: US Kentucky State Medicaid Terminations
```

Comment:
```suggestion
title: US Kentucky Terminated and Excluded Providers
```

Reply:
Let's try and align with the state's list name since that can make it easier for people looking for the list by its local name